### DISCUSSION TITLE
Color Buying/Selling text

### DISCUSSION THREAD

**Discussion by Cats1337** [QuickShop-Community/QuickShop-Hikari#1625] [Ideas]
On the signs, it isn't easy to see at a glance if a shop is buying or selling. I was looking through the config and saw no way of changing this. There's the option to change the sign text color, but if there was either the option or the default where Selling is green, and Buying is red. It would make it much easier to tell. This also comes with accessibility. Allowing people to customize the signs via the config, such as changing Selling to a symbol, or Unlimited to a Symbol and or moving where the item name is to be the second line, or last, etc.

**Comment by creatorfromhell** [comment]
> On the signs, it isn't easy to see at a glance if a shop is buying or selling. I was looking through the config and saw no way of changing this. There's the option to change the sign text color, but if there was either the option or the default where Selling is green, and Buying is red. It would make it much easier to tell. This also comes with accessibility. Allowing people to customize the signs via the config, such as changing Selling to a symbol, or Unlimited to a Symbol and or moving where the item name is to be the second line, or last, etc.

I guess the question is, does the "Selling text" not notate this enough?

**Reply by Cats1337** [reply]
Just having them colored so people can glance would help a lot, If you have a massive room full of shops, then going through and looking and reading each sign would be time consuming, if you could just look at see (green) or (red), you'd know it's selling or buying without having to read.

**Reply by creatorfromhell** [reply]
> Just having them colored so people can glance would help a lot, If you have a massive room full of shops, then going through and looking and reading each sign would be time consuming, if you could just look at see (green) or (red), you'd know it's selling or buying without having to read.

That's fair, I'll see what I can do for ya.

**Comment by creatorfromhell** [comment]
Sign customization is apart of 6.2.0.11 snapshots
